pkg/vpwned/server/vjailbreak_proxy.go

```diff
@@ -2,16 +2,15 @@ package server
 
 import (
 	"context"
-	"crypto/tls"
 	"fmt"
-	"net/http"
 	"strings"
 	"time"
 
-	gophercloud "github.com/gophercloud/gophercloud"
-	openstack "github.com/gophercloud/gophercloud/openstack"
-	ports "github.com/gophercloud/gophercloud/openstack/networking/v2/ports"
+	gophercloud "github.com/gophercloud/gophercloud/v2"
+	openstack "github.com/gophercloud/gophercloud/v2/openstack"
+	ports "github.com/gophercloud/gophercloud/v2/openstack/networking/v2/ports"
 	errors "github.com/pkg/errors"
+	netutils "github.com/platform9/vjailbreak/common/utils"
```

Comment:
these `v2` changes are already merged. Not sure why they are showing in the the diff